timesketch/lib/analyzers/tagger_test.py

```diff
@@ -0,0 +1,51 @@
+"""Tests for TaggerSketchPlugin."""
+import yaml
+
+from timesketch.lib import emojis
+from timesketch.lib.testlib import BaseTest
+
+
+class TestTaggerPlugin(BaseTest):
+    """Tests the functionality of the analyzer."""
+
+    def _config_validation(self, config):
+        """Validate that all items of a config are valid."""
+        query = config.get('query_string', config.get('query_dsl'))
+        self.assertIsNotNone(query)
+        self.assertIsInstance(query, str)
+
+        attribute = config.get('attribute')
```

Comment:
you can remove the attribute from the test as well, since there is no need to test something that isn't there

Reply:
fixed